bliss/simulator/prior.py

```diff
@@ -121,11 +126,12 @@ def __init__(
         self.galaxy_a_scale = galaxy_a_scale
         self.galaxy_a_bd_ratio = galaxy_a_bd_ratio
 
-    def sample_prior(self, batch_image_ids) -> TileCatalog:
-        """Samples latent variables from the prior of an astronomical image.
+        self.survey_data_dir = survey_data_dir
+        self.b_band = reference_band
+        self.gmm_star, self.gmm_gal = self._load_color_models()
 
-        Args:
-            batch_image_ids: image_ids needed for sampling.
+    def sample_prior(self) -> TileCatalog:
```

Comment:
Could we call this function `sample` rather than `sample_prior`, for consistency with the PyTorch distributions?